desktop/package.json

```diff
@@ -44,7 +44,7 @@
 		"enzyme": "^3.11.0",
 		"jest": "^27.3.1",
 		"lodash": "^4.17.21",
-		"playwright": "^1.18",
+		"playwright": "1.19",
```

Comment:
Any reason to not include the `^` here? It seems like we use the caret throughout Calypso. It shouldn't affect the version that gets installed because we use lockfiles, but it does make it easier to upgrade minor versions over time with `yarn upgrade`!

Reply:
Pushed updated commit to use caret.